woocommerce/woocommerce-admin
# wp.data Settings Refactor: update settings state

This PR changes the shape of settings state. Specifically, it removes the `group` property because changes introduced in #3430 make in not necessary.

New state has a shape like this.

```js
const state = {
	error, // error messages
	lastReceived // timestamp
	ids,  // array of data property names
	dirty, // array of property ids that have been changed without persistence
	[ id ]: { data, error, lastReceived }, // a sample data entry
}
```

### Testing Instructions

1. Load the settings page and make sure it renders
2. Change and save a settings
3. Make sure its working
